res/skins/LateNight/deck_small_row_1.xml

```diff
@@ -0,0 +1,130 @@
+                							Unicode, Arial, Verdana, sans-serif; text-align: left;}
+                						</Style>
+                						<Property>title</Property>
+                            <Elide>right</Elide>
+                            <SizePolicy>me,min</SizePolicy>
+                						<Channel>
+                							<Variable name="channum" />
+                						</Channel>
+                					</TrackProperty>
+                        </Children>
+                    </WidgetGroup>
+                    <WidgetGroup>
+                        <ObjectName>AlignRightTop</ObjectName>
+                        <Layout>horizontal</Layout>
+                        <SizePolicy>f,min</SizePolicy>
+                        <MinimumSize>100,</MinimumSize>
+                        <MaximumSize>100,</MaximumSize>
+                        <Children>
+                            <NumberPos>
+                                <ObjectName>TextColor<Variable name="channum" /></ObjectName>
+                                <TooltipId>track_time</TooltipId>
+                                <Alignment>right</Alignment>
+                                <Style>QLabel { font: bold 18px/21px Lucida Grande, Lucida Sans
+                                    Unicode, Arial, Verdana, sans-serif;}
+                                </Style>
+                                <Channel>
+                                    <Variable name="channum" />
+                                </Channel>
+                                <Connection>
+                                    <ConfigKey>[Channel<Variable name="channum" />],playposition</ConfigKey>
+                                </Connection>
+                            </NumberPos>
+                        </Children>
+                    </WidgetGroup>
+				</Children>
+			</WidgetGroup>-->
+			<WidgetGroup>
+				<ObjectName>ArtistRow</ObjectName>
+				<Layout>horizontal</Layout>
+				<SizePolicy>me,min</SizePolicy>
+				<Children>
+					<WidgetGroup>
+						<ObjectName>ArtistTitleSmall</ObjectName>
+						<Layout>horizontal</Layout>
+						<SizePolicy>me,min</SizePolicy>
+						<Children>
+							<TrackProperty>
+                                <ObjectName>TextColor<Variable name="channum" /></ObjectName>
+								<Alignment>Left</Alignment>
+								<TooltipId>track_artist</TooltipId>
+								<Style>QLabel { font: bold 13px/15px Lucida Grande, Lucida Sans
+									Unicode, Arial, Verdana, sans-serif; text-align: left;}
+								</Style>
+								<Property>artist</Property>
+								<!--<Elide>right</Elide>-->
+								<SizePolicy>min,min</SizePolicy>
+								<Channel>
+									<Variable name="channum" />
+								</Channel>
+							</TrackProperty>
+							<Label>
+								<ObjectName>TextColor<Variable name="channum" /></ObjectName>
+								<Style>QLabel { font: bold 13px/15px Lucida Grande, Lucida Sans
+									Unicode, Arial, Verdana, sans-serif; text-align: left;}</Style>
```

Comment:
Please add "Segoe UI" for windows users. font: lines should be consistent across whole skin.